### PR TITLE
translating tutorial/part-seven/index.md

### PR DESCRIPTION
**Pull request by rnd-debug**
If the PR is accepted, would it be possible to tag it with hacktoberfest-accepted? 😇 They have updated their policy in order to reduce spam PRs ([DigitalOcean link](https://hacktoberfest.digitalocean.com/hacktoberfest-update))

### CODE REVIEW

File: docs/tutorial/part-seven/index.md [hunk 1]
```diff
@@ -203,26 +195,27 @@ exports.createPages = async ({ graphql, actions }) => {
...
-pages.
+Vous avez ajouté une implementation de [`createPages`](/docs/node-apis/#createPages), API que Gatsby appelle pour ajouter des pages. 
 
-As mentioned in the intro to this part of the tutorial, the steps to programmatically creating pages are:
+Comme nous l'avons déjà dit dans l'introduction à ce tutoriel, pour créer des pages 
+de façon dynamique, il faut:
 
-1.  Query data with GraphQL
-2.  Map the query results to pages
+1.  Récupèrer les données avec GraphQL
```
Comment (dantereve) [inline docs/tutorial/part-seven/index.md]:
```suggestion
1.  Récupérer les données avec GraphQL
```

**Review by dantereve — DISMISSED** [review]
Another big page for you 🔥. Thanks for your help and rest assured that I will label your PRs with `hacktoberfest-accepted`. Hope you had fun in the process.

I've requested a microscopic change. Once this is done, let's merged this.

**Comment by rnd-debug** [comment]
Thx. Should be fixed :)

**Comment by mindsers** [comment]
This PR seems to be ready. Thanks for your work. I'll merge it! 🚀

**Review by mindsers — APPROVED** [review]
🚀

**Comment by rnd-debug** [comment]
@mindsers Thank you!

**Pull request merged**